src/Frontend.elm

```diff
@@ -923,19 +1010,19 @@ grantApplicationCopy =
     """
 If you would like to attend but are unsure about how to cover the combination of ticket and travel expenses, please get in touch with a brief paragraph about what motivates you to attend Elm Camp and how an opportunity grant could help.
 
-Please apply by sending an email to [team@elm.camp](mailto:team@elm.camp). The final date for applications is the 1st of May. Decisions will be communicated directly to each applicant by 5th of May. For this first edition of Elm Camp grant decisions will be made by Elm Camp organizers.
+Please apply by sending an email to [team@elm.camp](mailto:team@elm.camp). The final date for applications is the 1st of May. Decisions will be communicated directly to each applicant by 5th of May. Elm Camp grant decisions are made by the Elm Camp organizers using a blind selection process.
```

Comment:
```suggestion
Please apply by sending an email to [team@elm.camp](mailto:team@elm.camp). The final date for applications is the 1st of May. Decisions will be communicated directly to each applicant by 7th of May. Elm Camp grant decisions are made by the Elm Camp organizers using a blind selection process.
```

Reply:
DO we need to further clarify the meaning of 'blind selection process'?

Reply:
I don't think so – we didn't mention anything about the process last year, and we did an informal blind selection by virtue of stripping out personal / identifiable details before sharing with the rest of the team, so I thought it might be nice to mention that to further encourage applications with reduced fear of embarrassment.